### PR TITLE
Add 2.0 docs

### PR DESCRIPTION
**Pull request by freiksenet**

### CODE REVIEW

File: tools/graphql-tools/errors.md [hunk 1]
```diff
@@ -20,3 +20,4 @@ import { addErrorLoggingToSchema } from 'graphql-tools';
 const logger = { log: (e) => console.error(e.stack) };
 addErrorLoggingToSchema(mySchema, logger);
 ```
+dt
```
Comment (stubailo) [inline tools/graphql-tools/errors.md]: Extra text

**Comment by freiksenet** [comment]
@stubailo Done.

File: tools/graphql-tools/remote-schemas.md [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+<h3 id="fetcher-node-fetch" title="Using node-fetch">
+  Using [node-fetch](https://github.com/apollographql/node-fetch)
+</h3>
+
+Basic usage
+
+```js
+import fetch from 'node-fetch';
+
+const fetcher = async ({ query, variables, operationName, context }) => {
```
Comment (peggyrayzis) [inline tools/graphql-tools/remote-schemas.md]: For `node-fetch`, all you have to do is:
```
import fetch from 'node-fetch';

const link = new HttpLink({ uri: 'http://api.githunt.com/graphql', fetch });
```

We do need a way to pass that fetcher to `introspectSchema`. @jbaxleyiii what are your thoughts?

File: tools/graphql-tools/schema-stitching.md [hunk 3]
```diff
@@ -0,0 +1,311 @@
...
+import {
+  makeRemoteExecutableSchema,
+  introspectSchema,
+  mergeSchemas,
+} from 'graphql-tools';
+import { HttpLink, execute, makePromise } from 'apollo-link';
+
+async function makeMergedSchema() {
+  // Create remote executable schemas
+  const PropertyLink = new HttpLink({
```
Comment (peggyrayzis) [inline tools/graphql-tools/schema-stitching.md]: You have to pass the PropertyFetcher to HttpLink here.

**Pull request merged**